x/wasm-storage/keeper/abci.go

```diff
@@ -1,18 +1,48 @@
 package keeper
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
 	"cosmossdk.io/collections"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/sedaprotocol/seda-wasm-vm/tallyvm"
 )
 
+type Request struct {
+	// Commits           map[string]interface{} `json:"commits"`
+	DrBinaryID []byte `json:"dr_binary_id"`
+	// DrInputs          []interface{}          `json:"dr_inputs"`
+	GasLimit string `json:"gas_limit"`
+	GasPrice string `json:"gas_price"`
+	ID       []byte `json:"id"`
+	// Memo              []interface{}          `json:"memo"`
+	// PaybackAddress    []interface{}          `json:"payback_address"`
+	ReplicationFactor int                    `json:"replication_factor"`
+	Reveals           map[string]interface{} `json:"reveals"`
```

Comment:
Shouldn't we hard type this instead of doing a reveal?

```rust
pub struct RevealBody {
    pub salt:      String,
    pub exit_code: u8,
    pub gas_used:  U128, // string in json
    pub reveal:    Bytes, // base64 encoded string in json
}
```

Reply:
done in #280